spec/support/specialist_publisher_assertion_helpers.rb

```diff
@@ -47,11 +47,43 @@ def expect_discarded_draft
     expect(find(".alert").text).to match(/^Discarded/)
   end
 
+  def expect_attachment_removed
+    expect(find(".alert").text).to match(/^Attachment/)
+  end
+
   def expect_published_document
     expect(page).to have_content("There are no changes to publish.")
     expect_view_on_website_link
   end
 
+  def expect_add_attachment
+    within(".page-header")do
+      expect(page).to have_content("Add attachment")
+    end
+  end
+
+  def expect_attached_file_frontend
+    reload_page_while_failing do
+      within(".govuk-govspeak")do
+        expect(page).to have_link("Attached Doc")
+      end
+    end
+  end
+
+  def expect_removed_file_frontend
+    within(".govuk-govspeak")do
+      expect(page).to have_content("Removed attached document")
+    end
+  end
+
+  def expect_attached_file
+    expect(page).to have_content("1 attachment")
+  end
+
+  def expect_attached_file_removed
+    expect(page).to have_content("0 attachment")
```

Comment:
does it have the copy "0 attachment**s**" here that we can match off. Just would read slightly better - but understand the app could have a grammatical error.